ci-operator/step-registry/cucushift/installer/check/vsphere/disktype/cucushift-installer-check-vsphere-disktype-commands.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+source "${SHARED_DIR}/govc.sh"
+source "${SHARED_DIR}/vsphere_context.sh"
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+declare vsphere_datacenter
+declare vsphere_datastore
+INFRA_ID=$(jq -r .infraID ${SHARED_DIR}/metadata.json)
+check_result=0
+disk_type="${DISK_TYPE:-thin}"
+
+function check_disk_type() {
+    local ret=0 vm_lists disk_file vm_disk_type 
+    mapfile -t  vm_lists < <(govc ls /${vsphere_datacenter}/vm/${INFRA_ID})
+    if (( ${#vm_lists[@]} == 0 )); then
+        echo "vm_lists is empty, please check if vm still exits."
+    fi
+    for vm in "${vm_lists[@]}"; do
+        disk_file=$(govc vm.info -json /${vsphere_datacenter}/vm/${INFRA_ID}/${vm} | jq -r '.VirtualMachines[].Layout.Disk[].DiskFile[]' | awk  '{print $2}' | head -n1)
+	vm_disk_type=$(govc datastore.disk.info -ds /${vsphere_datacenter}/datastore/${vsphere_datastore} ${disk_file} | grep Type | awk -F ':    '  '{print $2}')
+	if [[ -n ${disk_type} ]];then
```

Comment:
we always defined `disk_type` with a default vaule, so we do not need this if condition.

Reply:
got it.